app/javascript/plugins/email_tool/EmailToolView.js

```diff
@@ -85,14 +93,37 @@ export default class EmailToolView extends Component {
     document.body.removeChild(htmlElement);
   };
 
-  handleCopyBodyButton = content => {
+  handleCopyBodyButton = () => {
     this.copyToClipboard(this.convertHtmlToPlainText(this.state.body));
     this.setState({ clickedCopyBodyButton: true });
   };
 
+  handleCopyTargetEmailButton = () => {
+    this.copyToClipboard(this.composeAllTargetEmails());
+  };
+
+  composeTargetAddresses(target) {
+    return `${join(compact([target.name, target.title]), ', ')} <${
+      target.email
+    }>`;
+  }
+
+  composeAllTargetEmails = () => {
+    let toEmailAddresses = null;
```

Comment:
why assign to `null`. Just leave it as undefined

Reply:
Fixed.